react/src/components/organisms/PageFlipper/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const PageFlipper = (props) => {
+  const contextLabel = (props.context.label) ? (<span className="ma__page-flipper__context-label">{props.context.label}</span>) : '';
+  const contextDecorativeLink = (props.context.DecorativeLink.props.text) ? props.context.DecorativeLink : '';
+  const contextLink = (contextLabel || contextDecorativeLink) ? (
+    <div className="ma__page-flipper__context">
+      {contextLabel}
+      {contextDecorativeLink}
+    </div>
+  ) : '';
+  const blank = (<div className="ma__page-flipper__blank">&nbsp;</div>);
+  const prev = props.PreviousLink ? props.PreviousLink : blank;
+  const next = props.NextLink ? props.NextLink : blank;
+  return (
+    <React.Fragment>
+      {contextLink}
+      <div className="ma__page-flipper">
+        <div className="ma__page-flipper__container">
+          {prev}
+          {next}
+        </div>
+      </div>
+    </React.Fragment>
+  );
+}
+
+PageFlipper.propTypes = {
+  context: PropTypes.shape({
+    label: PropTypes.string,
+    DecorativeLink: (props, propName, componentName) => {
```

Comment:
This prop shouldn't be the exact same name as a component, and it shouldn't be capitalized. There's also a utility function that covers validating a single component under src/components/utilities: componentPropTypeCheck

Reply:
updated